### ISSUE TITLE
Chrome throttling IPFS

### ISSUE THREAD

**Issue by ya7ya** [Paratii-Video/paratii-player#126]
[Chrome throttling background tabs](https://developers.google.com/web/updates/2017/03/background_tabs). Which affects the upload/download speed if the tab is in the background (not active).

**possible solution** : use ipfs in a service worker and see if that alleviates the issue. for reference there is an [`ipfs-service-worker`](https://github.com/ipfs/ipfs-service-worker) .

**Comment by milankragujevic** [comment]
Correct me if I'm wrong, but isn't service worker unable to use WebRTC, thereby having to use a proxy/bridge on a server to upload content..? ipfs/ipfs-service-worker uses the service worker to provide a download ability via ipfs, but doesn't allow uploading or WebRTC ... It's confusing to me, and I'm not sure if it can be solved easily. I want to propose a WebExtension-compatible extension for Chrome, Safari, Edge and Firefox that would operate the WebRTC processes independently of the throttling, but I'm not sure where that falls in regards to the philosophy of the project...

**Comment by ya7ya** [comment]
@milankragujevic Yeah, you are right, it seems you can't use `WebRTC` in the worker https://github.com/w3c/webrtc-pc/issues/230 . Thats a shame. Thanks for pointing that out dude 👍 

Regarding the WebExtension, I thought about it and it's a pretty good idea but we need this to work with zero installation required. But a WebExtension would be useful especially for uploaders who wanna seed files in the background i guess. I'll look more into it.

Thanks for your help clearing up this issue dude, I'll try to figure out some other solution beside service workers then.
